ROADMAP.md

```diff
@@ -0,0 +1,50 @@
+# Roadmap
+
+The freestyle history of the project was productive at its time.
+But recently, it has been outgrowing its freedom to freestyle.
+
+This roadmap gives an overview of the short and mid term direction that the project aims to take.
+
+## Deliverable Categories
+
+We've identified a couple of deliverable categories that should help us better understand the work spectrum associated with the project.
+Please also complement this with the high level overview in the [Architecture](./ARCHITECTURE.md).
+
+##### Process Categories alongside the SDLC
+
+- Setup Processes
+- The actual SDLC Cycle
+
+While an instance of the actual SDLC process inside the boundaries of Standard's current scope is easy to break down into _eval_, _build_ and _effect/action_ and easy to measure with the _cycle time_ metric.
```

Comment:
Seems like you were about to make a point but the sentence ends abruptly.

Reply:
colon instead of fullstop :smile: